src/main/groovy/com/homeaway/devtools/jenkins/testing/JenkinsPipelineSpecification.groovy

```diff
@@ -735,7 +742,11 @@ public abstract class JenkinsPipelineSpecification extends Specification {
 		
 		return script
 	}
-	
+
+	private String[] generateScriptClasspath(String resourcePath) {
```

Comment:
Please make this `protected`, not `private`.

Reply:
I updated the PR. Thanks for the fast review 👍